### PR TITLE
[Doc] add-note-for-auto-bucket-number-setting

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: docs/table_design/Data_distribution.md [hunk 1]
```diff
@@ -547,6 +547,10 @@ Buckets reflect how data files are actually organized in StarRocks.
 
     To enable this feature, make sure that the FE dynamic parameter `enable_auto_tablet_distribution` is set to `TRUE`. After a table is created, you can execute [SHOW CREATE TABLE](../sql-reference/sql-statements/data-manipulation/SHOW%20CREATE%20VIEW.md) to view the bucket number automatically set by StarRocks.
 
+    > **NOTICE**
+    >
+    > If the raw data size of a partition exceeds 100GB, we recommend that you manually configure the number of buckets using the following method.
```
Comment (amber-create) [inline docs/table_design/Data_distribution.md:552]: image
1）-> 100 MB
2) -> the following Method 2 OR Method 2 described below. (I think "Method 2" is easier to find.)

**Review by amber-create — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: docs/table_design/Data_distribution.md [hunk 2]
```diff
@@ -547,6 +547,10 @@ Buckets reflect how data files are actually organized in StarRocks.
 
     To enable this feature, make sure that the FE dynamic parameter `enable_auto_tablet_distribution` is set to `TRUE`. After a table is created, you can execute [SHOW CREATE TABLE](../sql-reference/sql-statements/data-manipulation/SHOW%20CREATE%20VIEW.md) to view the bucket number automatically set by StarRocks.
 
+    > **NOTICE**
+    >
+    > If the raw data size of a partition exceeds 100 GB, we recommend that you manually configure the number of buckets using the following Method 2.
```
Comment (evelynzhaojie) [inline docs/table_design/Data_distribution.md:552]: using the following Method 2 --- using Method 2

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by evelynzhaojie — APPROVED** [review]

**Review by EsoragotoSpirit — APPROVED** [review]

**Comment by hellolilyliuyi** [comment]
@mergify backport branch-3.1 branch-3.0 branch-2.5

**Pull request merged**